repos/system_upgrade/el7toel8/libraries/config/version.py

```diff
@@ -0,0 +1,113 @@
+import operator
+
+import six
+
+from leapp.libraries.stdlib import api
+
+
+OP_MAP = {
+    '>': operator.gt,
+    '>=': operator.ge,
+    '<': operator.lt,
+    '<=': operator.le
+}
+
+
+def _version_to_tuple(version):
+    """Converts the version string ``major.minor`` to ``(major, minor)`` int tuple."""
+    major, minor = version.split('.')
+    return (int(major), int(minor))
+
+
+def _validate_versions(versions):
```

Comment:
You don't need to do the validation explicitly, and just catch ValueError in _version_to_tuple and transform the error to the exception below

Reply:
The function `_validate_versions` is used in multiple places and catching the `ValueError` exception in `_version_to_tuple` would work only for few cases, for which `_validate_versions` is used.